test/Test/Cardano/Prelude/GHC/Heap/NormalForm/Classy.hs

```diff
@@ -51,6 +50,11 @@ class (NoUnexpectedThunks a, Show (Model a)) => FromModel a where
   -- | Does the model describe a value in NF?
   modelIsNF :: [String] -> Model a -> IsNormalForm [String]
 
+  -- | Convert the model to one that represents its equivalent in weak head
+  -- normal form.
+  toWHNFModel :: Model a -> Model a
+  toWHNFModel m = m
```

Comment:
@edsko: Curious about your thoughts on this addition. This assists me in the `ListForceElems` case of `modelIsNF`. 

Since the `ListForceElems` case of `fromModel` uses `forceElemsToWHNF`, the resulting implementation value consists of elements that are all in WHNF. However, the model itself might not necessarily represent a Haskell value whose elements are all in WHNF. Because of this, we need some way to convert/"force" each element of the `ListForceElems` to what would be its WHNF representation. If we don't do this, the model and implementation won't agree on the thunk info as the model could actually represent something "more thunky" than the actual implementation value.

Reply:
Yes, I think it makes sense. It might also simplify the instances for `a -> b`.

Reply:
Adding these changes as part of https://github.com/input-output-hk/cardano-prelude/pull/78 instead.